pkg/getty/root/.profile

```diff
@@ -0,0 +1 @@
+PS1="(getty ns) \w \$ "
```

Comment:
Would you mind doing the same to the sshd container too?

Perhaps we should put this in `/etc/profile` or some other suitable global file, since the `getty` container (at least in theory, I've not tried it) supports non-root logins. It looks like alpine's `/etc/profile` sources `/etc/profile.d/*.sh` which makes it nice and easy to drop a file in.

Debian's default profile automatically adds the contents of `/etc/debian_chroot` to `$PS1` (which can be usefully abused for non-chroot purposes too!), although `/etc/profile.d/*.sh` makes it mostly moot I suppose.